ios-shoppe-demo/ios-shoppe-demo.xcodeproj/xcuserdata/harold.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
```

Comment:
@sabrina-li mentioned this in a prior review: can you .gitignore the `xcuserdata` directory?

Reply:
I originally added it to the .gitignore. We agreed it would be safer just to leave it for now as i will do some more research on this in the near future. Unless you think it's a priority now, if so I can give it my attention right away for this commit, I would like to push this forward.

Reply:
Hey @HaroldFS,

> We agreed it would be safer just to leave it for now

I'd love to know more about this.

> Unless you think it's a priority now, if so I can give it my attention right away for this commit

Yes, I can see issues in the future. I'm not sure what the implications are if someone were to clone this repo into their own dev environment and then try to run it with your workspace configs committed.

There will definitely be an issue in the future if someone wanted to contribute to this project. They would have to deal with excluding their own workspace config or accidentally committing it, then we'd need to handle it in the PR. I doubt we'd want multiple workspace configs from our contributors in this repo.

Please let me know your thoughts.

Reply:
I'm pretty positive (99.9% sure) there are no issues. Xcode auto reconfigures this based on the user but i understand its unnecessary to have this info show up in a PR I'll do the research and get this done right away I really want to move past this initial commit today if possible.

Reply:
(For your first question to know more) About why i thought it was safer is just simply if I dont know exactly why i wanted to remove it i figured it was safer not to do so right off the bat. And make a conscious decision later. (when i better knew what I was doing on the matter).
 
for the second comment no worries I believe I just found the specific solution and I'm implementing it now. I really would like to get this knocked out.

Reply:
I think the following examples are going to be helpful on figuring out what's needed here:
I would def add DS_Store, build files, pods, and userdata, this feels like a good reference:
https://gist.github.com/bojan/8706331
https://stuartbreckenridge.com/default-gitignore/ (kinda old)